### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by flocke** [leinardi/FloatingActionButtonSpeedDial#149]
I use the SpeedDial in my project (https://github.com/andOTP/andOTP) and a user recently opened an issue regarding the accessibility of the app which included some comments regarding the FAB menu. I was able to fix some of them myself, but there are still some things left which should be handled by the menu itself:

* When the FAB is pressed and the overlay shown:
    - One of the options displayed should be focused;
    - The user should not be able to swipe to any of the user interface elements that are hidden behind the overlay;
    - Focusing the FAB or overlay should indicate that activating will close the overlay, whereas currently the FAB is unlabelled and the overlay reports nothing.
    - The smaller FAB buttons of the menu should have their content description set to the value of the label

You can find the original issue here: https://github.com/andOTP/andOTP/issues/498

**Comment by leinardi** [comment]
Hi @flocke, I should have fixed all the issues (or provided a way to fix them downstream). Could you please have a look at the PR and let me know what you think?

**Comment by flocke** [comment]
@leinardi Thank you for implementing this, I have to apologize for being so slow to review it. I must admit that I totally forgot about this issue...

I took your changes for a spin in andOTP and it seems like they are working nicely.

There are only two things I found that could be problematic:

1. If you focus the overlay there is no content description, it can be set downstream by calling the `setContentDescription()` function on the overlay. But maybe a default description for the overlay would be nice (maybe the same as the "Close menu" button, since clicking it has the same function of dismissing the menu).
2. I managed to swipe "behind the overlay" after focusing it and access the content of the app behind it while the menu is still open. We should either prevent that, or close the menu when that happens.

**Comment by fei0316** [comment]
Any updates on when this will be included in a release?

**Comment by leinardi** [comment]
Hi @fei0316 I'm trying to have a new release over this weekend.

**Comment by leinardi** [comment]
Release 3.2.0 is available on Maven Central: https://repo1.maven.org/maven2/com/leinardi/android/speed-dial/maven-metadata.xml
